docs/internals/contributing/writing-documentation.txt

```diff
@@ -18,54 +18,89 @@ Documentation changes generally come in two forms:
 This section explains how writers can craft their documentation changes
 in the most useful and least error-prone ways.
 
-Getting the raw documentation
-=============================
+How to start writing documentation
+==================================
 
-Though Django's documentation is intended to be read as HTML at
-https://docs.djangoproject.com/, we edit it as a collection of text files for
-maximum flexibility. These files live in the top-level :source:`docs/` directory of a
-Django release.
+Clone the Django repository to your local machine
+-------------------------------------------------
 
 If you'd like to start contributing to our docs, get the development version of
-Django from the source code repository
-(see :ref:`installing-development-version`). The development version has the
-latest-and-greatest documentation, just as it has the latest-and-greatest code.
-We also backport documentation fixes and improvements, at the discretion of the
-merger, to the last release branch. That's because it's highly advantageous to
-have the docs for the last release be up-to-date and correct (see
-:ref:`differences-between-doc-versions`).
+Django from the source code repository (see
+:ref:`installing-development-version`):
 
-Getting started with Sphinx
-===========================
+.. console::
 
-Django's documentation uses the Sphinx__ documentation system, which in turn
-is based on docutils__. The basic idea is that lightly-formatted plain-text
-documentation is transformed into HTML, PDF, and any other output format.
+     $ git clone https://github.com/django/django.git
 
-__ https://www.sphinx-doc.org/
-__ https://docutils.sourceforge.io/
+If you're planning to submit these changes, you might find it useful to make a
+fork of the Django repository and clone this fork instead.
 
-To build the documentation locally, install Sphinx:
+Set up a virtual environment and install dependencies
+-----------------------------------------------------
+
+Create and activate a virtual environment, then install the dependencies:
 
 .. console::
 
-     $ python -m pip install Sphinx
+     $ python -m venv .venv
+     $ source .venv/bin/activate
+     $ python -m pip install -r docs/requirements.txt
```

Comment:
Is there a reason to copy here a part of `installing-development-version` and a part of Python docs? instead of leaving references to them, as it was.

Reply:
Good question, but I think that in the conversations over the last two days we have identified a general problem that contributors - especially new ones - find themselves opening tab after tab when they follow references to things like that, and this will help as a first step to streamlining their journey, so that there is less risk of forking and getting lost in the workflow.